web/public/index.html

```diff
@@ -8,10 +8,10 @@
 	<title>Driving Log</title>
 
 	<!-- CSS -->
-	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+	<link href="http://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
this has to stay https or else the request will be blocked (it's weird like that)